ee/zentral/contrib/intune/models.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from django.db import models
+from django.db.models import F
+from django.urls import reverse
+
+from zentral.core.secret_engines import decrypt_str, encrypt_str, rewrap
+
+
+logger = logging.getLogger("zentral.contrib.intune.models")
+
+
+class Tenant(models.Model):
+    business_unit = models.ForeignKey("inventory.BusinessUnit", on_delete=models.PROTECT)
+    name = models.CharField(max_length=256, unique=True)
+    description = models.TextField(blank=True, null=True)
```

Comment:
TextFields and CharFields are better without the null=True, to avoid having a double empty value, null or "". See [the Django docs](https://docs.djangoproject.com/en/4.2/ref/models/fields/#null) for more context.